### ISSUE TITLE
Support for AIX platfrom

### ISSUE THREAD

**Issue by sureshbkn** [danielaparker/jsoncons#151]
Is the library tested for IBM AIX platform, intend to use it with IBM AIX latest compiler 13.1.3.

**Comment by danielaparker** [comment]
No, we currently only test on intel platforms, with Linux and Windows. It should compile with any C++ 11 compliant compiler. If you go ahead, I would suggest running the test suite, note that the test suite currently has a dependency on boost. If you experience any issues, please report them, and we'll do our best to address them.
